BSP/Simulator/Makefile

```diff
@@ -61,7 +61,8 @@ FLAGS = -Wall -g -std=c11 $(INC_DIR)
 ######################################
 # Defines
 ######################################
-DEF = -D_XOPEN_SOURCE=600 -DSIMULATION
+DEF = -D_XOPEN_SOURCE=700 -DSIMULATION
+DEF = -D_POSIX_C_SOURCE=200809L
```

Comment:
```suggestion
DEF += -D_POSIX_C_SOURCE=200809L
```

Reply:
What does this do?

Reply:
It adds...

Right now you are essentially doing (in pseudo-C):
```c
string DEF = "-D_XOPEN_SOURCE=700 -DSIMULATION"
string DEF = "-D_POSIX_C_SOURCE=200809L"
```
So at the end, DEF is just `"-D_POSIX_C_SOURCE=200809L"`.

If you append rather than reassign, it will end as `"-D_XOPEN_SOURCE=700 -DSIMULATION -D_POSIX_C_SOURCE=200809L"`, which is what I assume you meant to do.